src/routes/+page.svelte

```diff
@@ -8,6 +8,8 @@
 	import AnimatedCarousel from '$lib/components/AnimatedCarousel/AnimatedCarousel.svelte'
 	import Text from '$lib/components/Text.svelte'
   import Button from '$lib/components/Button/Button.svelte'
+  import Toast from '$lib/components/Toast/Toast.svelte'
```

Comment:
Weird indentation? 🤔

Reply:
this must be a GH glitch, I'm not seeing any formatting issues